tfe/resource_tfe_organization_token.go

```diff
@@ -41,6 +41,12 @@ func resourceTFEOrganizationToken() *schema.Resource {
 				Computed:  true,
 				Sensitive: true,
 			},
+
+			"expired_at": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
❓ why is this computed? My understanding of computed is that it is for properties which cant be known before the object is created.

https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed